spec/factories/playbooks.rb

```diff
@@ -1,16 +1,8 @@
 # frozen_string_literal: true
 
-# == Schema Information
```

Comment:
Can we restore this schema info?

Reply:
There appeared to be several files that needed documentation updates.  I ran the annotate gem and included them all in commit dc1a1f1.  I think it might be worth considering adding a task to the rakefile to automate that process whenever a new migration is run. 

From the gem documentation:
![Screen Shot 2020-06-01 at 11 02 51 AM](https://user-images.githubusercontent.com/20918543/83422429-857c8000-a3f7-11ea-8753-e9aa3fe4ee9b.png)

Reply:
Added issue #132 to automate documentation.